README.md

```diff
@@ -19,6 +19,88 @@ The operator was developed/tested for only:
 
 # Run
 
+## User Permissions
+To install the percona-server-mongodb-operator please ensure that you have enough permission. 
+You'll need to have cluster-admin role to install operator, also you can grant to specified user additional permissions by applying next policy:
+```yaml
```

Comment:
Is this YAML blob the same as deploy/rbac.yaml? If so these steps are ran automatically in the 'kubectl create -f depoy/rbac.yaml' step further down in the README